src/app/app.component.html

```diff
@@ -2,7 +2,7 @@
   <ion-menu contentId="main-content" type="overlay" side="end" maxEdgeStart="0">
     <app-side-menu></app-side-menu>
   </ion-menu>
-  <ion-menu side="start" menuId="filter" contentId="main-content">
+  <ion-menu [swipeGesture]="swipeBackEnabled$ | async" side="start" menuId="filter" contentId="main-content">
```

Comment:
Jeg tror egentlig det holder å sette `maxEdgeStart="0"` her, som for den andre menyen. Da slipper vi også endringene i de andre filene.